src/App.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import "./App.css";
+
```

Comment:
would have loved to see you use ACE editor! really good code tho!

Reply:
I did not know ACE, and it cost me a little to understand the exercise. The next time I know of its existence and I will use it. Thank you